Microsoft.Research/Contracts/PresentationFramework/Sources/System.Windows.Controls.Panel.cs

```diff
@@ -43,10 +43,11 @@ abstract public partial class Panel : System.Windows.FrameworkElement, System.Wi
     #region Methods and constructors
     protected virtual new UIElementCollection CreateUIElementCollection(System.Windows.FrameworkElement logicalParent)
     {
-      return default(UIElementCollection);
+        Contract.Ensures(Contract.Result<UIElementCollection>() != null);
+        return default(UIElementCollection);
     }
 
-    protected override System.Windows.Media.Visual GetVisualChild(int index)
+      protected override System.Windows.Media.Visual GetVisualChild(int index)

```

Comment:
I think you should feel free to change the convention to 4. Maybe just open an issue to ask for a vote. But it would be easier for everyone if it matched the VS standard.

Mike

Sent from my Windows Phone
________________________________
From: tom-englert<mailto:notifications@github.com>
Sent: ý3/ý16/ý2015 2:07 AM
To: Microsoft/CodeContracts<mailto:CodeContracts@noreply.github.com>
Subject: Re: [CodeContracts] Add some missing system contracts; fix bad contracts in Application and ... (#21)


In Microsoft.Research/Contracts/PresentationFramework/Sources/System.Windows.Controls.Panel.cs<https://github.com/Microsoft/CodeContracts/pull/21#discussion_r26468792>:

>      }
>
> -    protected override System.Windows.Media.Visual GetVisualChild(int index)
> +      protected override System.Windows.Media.Visual GetVisualChild(int index)


Sorry, it's hard to keep the old formatting with 2 spaces indent where the default in VS is 4 spaces. Are there any coding guidelines for CC project?

—
Reply to this email directly or view it on GitHub<https://github.com/Microsoft/CodeContracts/pull/21/files#r26468792>.